pkgs/by-name/ge/geteduroam/package.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  symlinkJoin,
+  versionCheckHook,
+  makeWrapper,
+  cairo,
+  gdk-pixbuf,
+  glib,
+  graphene,
+  gtk4,
+  libadwaita,
+  pango,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "geteduroam";
+  version = "0.10";
+
+  src = fetchFromGitHub {
+    owner = "geteduroam";
+    repo = "linux-app";
+    tag = finalAttrs.version;
+    hash = "sha256-Mtzt6i8vJ5M8T0vrAOxXhawlhCmCMEnDQz0Jo6uV88A=";
+  };
+
+  vendorHash = "sha256-b06wnqT88J7etNTFJ6nE9Uo0gOQOGvvs0vPNnJr6r4Q=";
+
+  subPackages = [
+    "cmd/geteduroam-gui"
+    "cmd/geteduroam-notifcheck"
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/geteduroam-gui \
+      --prefix PUREGOTK_LIB_FOLDER : ${finalAttrs.passthru.libraryPath}/lib
```

Comment:
Can this env var actually be a list of colon-separated entries?
If not, you should use `--set` or `--set-default`

Reply:
good catch, copy-pasted logic error from vinegar 
* https://github.com/NixOS/nixpkgs/pull/408210